config.properties

```diff
@@ -0,0 +1 @@
+db_url=jdbc:mysql://localhost/hotel?useTimeZone=true&serverTimeZone=UTC
```

Comment:
Please rename this Pull Request from `Temp. properties file (URL) and environment variables (Password and` so something related to the purpose of this changes.